pkgs/development/python-modules/catboost/default.nix

```diff
@@ -1,38 +1,38 @@
 { buildPythonPackage, fetchFromGitHub, fetchpatch, lib, pythonOlder
-, clang_7, python2
+, clang_12, python, python2, sd
```

Comment:
Can we finally get rid of python2?

Reply:
Most likely not, but it is only a build dep.